src/ui-kit/button.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { Button as AntdButton } from 'antd'
+
+const Button = ({ title }) => (
+  <span>
+    <AntdButton style={{ fontSize: '25px' }}>{title}</AntdButton>
+  </span>
+)
+Button.propTypes = {
```

Comment:
this is not readable at all..put an empty line between

Reply:
```
const Button = ({ title }) => (
  <span>
    <AntdButton style={{ fontSize: '25px' }}>{title}</AntdButton>
  </span>
)

Button.propTypes = {...}
```

Reply:
also you can directly import the function you like from prop-types, the last version of prop-types supports that:

**import { func } from 'prop-types'**
```
Button.propTypes = {
  func.isRequired
}
```

Reply:
I understand what you want say, but i don't find another example: ` import { func } from 'prop-types'`